avae/train.py

```diff
@@ -122,6 +134,9 @@ def train(
         eval=False,
         affinity=affinity,
         classes=classes,
+        gaussian_blur=gaussian_blur,
+        normalise=normalise,
+        shift_min=shift_min,
```

Comment:
Add to shift-mean to docstring.

Reply:
fixed in [5876b00](https://github.com/alan-turing-institute/affinity-vae/pull/144/commits/5876b00ca5e0015fecbeed2d7376a3eb90f58af9)